sycl/doc/extensions/GroupAlgorithms/SYCL_INTEL_group_algorithms.asciidoc

```diff
@@ -51,7 +51,9 @@ John Pennycook, Intel (john 'dot' pennycook 'at' intel 'dot' com)
 
 == Dependencies
 
-This extension is written against the SYCL 1.2.1 specification, Revision v1.2.1-6.
+This extension is written against the SYCL 1.2.1 specification, Revision v1.2.1-6, the SYCL 2020 provisional specification and the following extensions:
```

Comment:
This is probably a minor-medium comment, but strictly speaking, we should be careful with listing two specifications here at the same time:

Both SYCL 1.2.1 and SYCL 2020 provisional are listed under "Dependencies" title. This might be confusing, I think, as specs are not fully backward compatible: for example there is no `program` class in SYCL 2020 provisional. This fact is not really related to this particular extension, but in general might cause some inconsistencies in extension spec, I guess.

`SYCL_INTEL_extended_atomics` which is listed in description, is built upon SYCL 1.2.1 and it is unclear whether it is needed or not, if this extension applied to SYCL 2020 provisional.

Usually, extension refers to the spec it extends in order to replace/extend/modify particular sections from there. This might be tricky if you refer to two specs at the same time. For example. this proposal has the following reference:

> All functions are supported for the fundamental scalar types supported by SYCL and instances of the SYCL +vec+ class. The fundamental scalar types (as defined in Section 6.5 of the SYCL 1.2.1 specification)

However, the same table in SYCL 2020 provisional spec is located in different section and theoretically, might contain different list of types.

This comment is not intended to block the commit, but I think that we should properly write our extensions so it is clear to which existing spec/functionality they refer and how they change it

Reply:
This is a fair comment, and I think you're right that this should be changed.  The only reason SYCL 2020 provisional is mentioned here is because I wanted to make it clear that the new `barrier` function is based on the `group_barrier` functionality from that specification.

I'll change this to say that the extension is written against SYCL 1.2.1, and add a separate note about `group_barrier` as an attribution rather than a dependency.

Reply:
Updated version looks better, IMHO, thanks!